{
  "comments": [
    {
      "key": {
        "uuid": "d3682185_7f0465fc",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 11,
      "author": {
        "id": 1012732
      },
      "writtenOn": "2020-04-27T17:55:01Z",
      "side": 1,
      "message": "Is this also what the exception message\u0027s reason part was? If not, maybe revisit the decision of not logging the exception reason.\n\nIf the exception reason had more information than just that string, the exception message (uri and reason parts) could be logged and not the uri alone.",
      "range": {
        "startLine": 11,
        "startChar": 0,
        "endLine": 11,
        "endChar": 14
      },
      "revId": "3cf056131792a031e80476efda35f4b43315d546",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153",
      "unresolved": true
    }
  ]
}